nebula/ui/components/VPNLinkButton.qml

```diff
@@ -106,28 +106,38 @@ VPNButtonBase {
 
     VPNMouseArea {
         id: buttonMouseArea
-        hoverEnabled: loaderVisible === false
+        hoverEnabled: loaderVisible === false && parent.enabled
     }
 
     background: Rectangle {
         id: backgroundRect
         color: VPNTheme.theme.transparent
     }
 
-    contentItem: Label {
-        id: label
-
-        text: labelText
-        color: root.linkColor.defaultColor
-        horizontalAlignment: textAlignment
-        verticalAlignment: Text.AlignVCenter
-        font.pixelSize: fontSize
-        font.family: fontName
-        wrapMode: Text.WordWrap
-        opacity: loaderVisible ? 0 : 1
-        Behavior on color {
-            ColorAnimation {
-                duration: 200
+    contentItem: RowLayout {
+        spacing: VPNTheme.theme.windowMargin / 2
+
+        Loader {
+            id: iconLoader
+            Layout.alignment: Qt.AlignVCenter
+            sourceComponent: iconComponent
+        }
+
+        Label {
+            id: label
+
+            text: labelText
+            color: root.linkColor.defaultColor
+            horizontalAlignment: textAlignment
+            verticalAlignment: Text.AlignVCenter
+            font.pixelSize: fontSize
+            font.family: fontName
+            wrapMode: Text.WordWrap
+            opacity: loaderVisible ? 0 : 1
+            Behavior on color {
+                ColorAnimation {
+                    duration: 200
+                }
             }
         }
```

Comment:
This seems to be causing a layout issue in VPNTutorialPopup (and possibly other places to). Could maybe try adding `Layout.alignment: Qt.AlignHCenter` to the loader and the label?

<img width="428" alt="Screen Shot 2023-02-13 at 11 37 01 PM" src="https://user-images.githubusercontent.com/15353801/218640582-9c539f04-a7ef-4982-b0d6-3e5ce4bd007c.png">

Reply:
oh good catch. fixing.

Reply:
Ok! So the issue was we were stipulating `Layout.fillWidth: true` for these buttons in VPNSimplePopUp. For the primary button that makes sense, but for VPNLinkButton this creates a huge clickable, though invisible, area on either side of the button.

VPNSimplePopUp screen caps, post-changes:
<details>
<img width="300" alt="Screen Shot 2023-02-14 at 10 58 32 AM" src="https://user-images.githubusercontent.com/22355127/218806996-1d92727c-72b1-4729-aac3-7be1a1ccbec0.png">
<img width="300" alt="Screen Shot 2023-02-14 at 10 52 16 AM" src="https://user-images.githubusercontent.com/22355127/218807034-b955ed7e-bf0e-48b4-9e17-e9d3f950f738.png">
<img width="300" alt="Screen Shot 2023-02-14 at 10 51 53 AM" src="https://user-images.githubusercontent.com/22355127/218807052-eb68417b-686e-4113-84d5-2b8ba5b99cfd.png">
<img width="300" alt="Screen Shot 2023-02-14 at 10 49 37 AM" src="https://user-images.githubusercontent.com/22355127/218807076-5415ffe8-4daf-49f8-9bb7-c9cc4d04da5d.png">
<img width="300" alt="Screen Shot 2023-02-14 at 10 46 10 AM" src="https://user-images.githubusercontent.com/22355127/218807220-efc6de91-42ff-4ae1-a183-ce9ba56b498a.png">
<img width="300" alt="Screen Shot 2023-02-14 at 10 36 41 AM" src="https://user-images.githubusercontent.com/22355127/218807247-c9592dd9-383a-4c48-85d4-ffa446a7cee8.png">
<img width="300" alt="Screen Shot 2023-02-14 at 10 34 34 AM" src="https://user-images.githubusercontent.com/22355127/218807328-2668a51a-d30d-4d5f-bbf7-5ef6a99a750d.png">
<img width="300" alt="Screen Shot 2023-02-14 at 10 28 37 AM" src="https://user-images.githubusercontent.com/22355127/218807367-cfe8c816-61f1-46ef-a076-b7a1800e95cc.png">
</details>